Error during building examples for ARC target in TFLM.

@tensorflow/micro

**System information**
- Host OS Platform and Distribution (e.g., Linux Ubuntu 16.04): all
- TensorFlow installed from (source or binary): source
- Tensorflow version (commit SHA if source): master
- Target platform (e.g. Arm Mbed OS, Arduino Nano 33 etc.): ARC

During building generated for ARC target project, user gets the following error:
```
In file included from tensorflow/lite/micro/kernels/arc_mli/fully_connected.cc:27:
.\tensorflow/lite/micro/kernels/arc_mli/mli_tf_utils.h:99:16: error: static_cast from 'const int *' to 'void *' is not allowed
  mliT->data = static_cast<void*>(tflite::micro::GetTensorData<datatype>(tfT));
               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Need to fix mli_tf_utils.h file.

--- a/tensorflow/lite/micro/kernels/arc_mli/mli_tf_utils.h
+++ b/tensorflow/lite/micro/kernels/arc_mli/mli_tf_utils.h
@@ -96,7 +96,11 @@ inline void ConvertToMliQuantParamsPerChannel(const TfLiteTensor* tfT,
 template <typename datatype>
 inline void MliTensorAttachBuffer(const TfLiteEvalTensor* tfT,
                                   mli_tensor* mliT) {
-  mliT->data = static_cast<void*>(tflite::micro::GetTensorData<datatype>(tfT));
+  // "const_cast" here used to attach const data buffer to the initially
+  // non-const mli_tensor. This is required by current implementation of MLI
+  // backend and planned for redesign due to this and some other aspects.
+  mliT->data = const_cast<void*>(
+      static_cast<const void*>(tflite::micro::GetTensorData<datatype>(tfT)));
 }
 
 inline void ConvertToMliTensor(const TfLiteTensor* tfT, mli_tensor* mliT) {

